ext/mixed/js/display-generator.js

```diff
@@ -258,6 +276,117 @@ class DisplayGenerator {
         return node;
     }
 
+    createPitches(details) {
+        if (!this._termPitchAccentStaticTemplateIsSetup) {
+            this._termPitchAccentStaticTemplateIsSetup = true;
+            const t = this._templateHandler.instantiate('term-pitch-accent-static');
+            document.head.appendChild(t);
+        }
```

Comment:
Is this worth the optimization or are there other concerns? Maybe it could be added in `prepare`.

Reply:
In cases where pitch accent dictionaries aren't installed, it won't be added. I suppose it's not a big deal either way if it's unused, so it could go into prepare.

Reply:
Found this old review comment after reviewing the history, feel free to move it there now or leave as is because it's just a minor code style issue.